subsys/mgmt/updatehub/updatehub_integrity.c

```diff
@@ -9,11 +9,7 @@ LOG_MODULE_DECLARE(updatehub, CONFIG_UPDATEHUB_LOG_LEVEL);
 
 #include "updatehub_integrity.h"
 
-#if defined(CONFIG_PSA_CRYPTO_CLIENT)
 #define SUCCESS_VALUE PSA_SUCCESS
```

Comment:
seems like this intermediate define is no longer needed and PSA_SUCCESS could be used directly

Reply:
oh yes, somehow i didn't saw that